Atomic/backendutils.py

```diff
@@ -41,9 +41,12 @@ def get_backend_and_image_obj(self, img, str_preferred_backend=None):
         # Check preferred backend first
         if str_preferred_backend:
             be = self.get_backend_from_string(str_preferred_backend)
-            img_obj = be.has_image(img)
-            if img_obj:
-                return be, img_obj
+            try:
```

Comment:
How about we pass an option to the be saying that it should ignore nodaemon errors?  I don't want to accidently eat errors when we want them reported to the user?